### PR TITLE
Create bugreport.yml

### PR DESCRIPTION
**Pull request by CottageDwellingCat**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this bug report!
+  - type: checkboxes
+    attributes:
+      label: Check The Docs
+      description: Please refer to our [FAQs](https://discordnet.dev/faq), [Documentation](https://discordnet.dev/api/index.html), 
+                    and [Migration Guide](https://discordnet.dev/guides/v2_v3_guide/v2_to_v3_guide.html) before reporting issues.
+      options:
+      - label: "I double checked the docs and couldn't find amy useful information."
```
Comment (lhjt) [inline .github/ISSUE_TEMPLATE/bugreport.yml:15]:
```suggestion
      - label: "I double checked the docs and couldn't find any useful information."
```

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        required: true
+  - type: checkboxes
+    attributes:
+      label: Verify Issue Source
+      description: If your issue is related to an exception make sure the error was thrown by Discord.Net, and not your code or another library. 
+                  If you get an `HttpException` with the error code `401`, than the error is caused by your bots permissions, not dnet.
```
Comment (lhjt) [inline .github/ISSUE_TEMPLATE/bugreport.yml:21]:
```suggestion
                  If you get an `HttpException` with the error code `401`, then the error is caused by your bot's permissions, not dnet.
```

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+      options:
+      - label: I verified the issue was caused by Discord.Net.
+        required: true
+  - type: checkboxes
+    attributes:
+      label: Check your intents
+      description: If your issue is related to not recieving expected events you may have setup your gateway intents incorectly. 
+                    Newer versions of Discord.Net use a more modern versions of Discords API that requires you tell it what events
+                    you want to recieve. Discord.Net defaults to all non-privleged intents, but if your bot requires privleged intents
```
Comment (lhjt) [inline .github/ISSUE_TEMPLATE/bugreport.yml:30]:
```suggestion
      description: If your issue is related to not receiving expected events, you may have setup your gateway intents incorrectly. 
                    Newer versions of Discord.Net use a more modern version of Discord's API that requires you tell it what events
                    you want to receive. Discord.Net defaults to all non-privleged intents, but if your bot requires privileged intents 
```

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+      - label: I verified the issue was caused by Discord.Net.
+        required: true
+  - type: checkboxes
+    attributes:
+      label: Check your intents
+      description: If your issue is related to not recieving expected events you may have setup your gateway intents incorectly. 
+                    Newer versions of Discord.Net use a more modern versions of Discords API that requires you tell it what events
+                    you want to recieve. Discord.Net defaults to all non-privleged intents, but if your bot requires privleged intents 
+                    you need specify them in your clients config. You can see what intents you need for your events 
+                    [here](https://discord.com/developers/docs/topics/gateway#list-of-intents)
```
Comment (lhjt) [inline .github/ISSUE_TEMPLATE/bugreport.yml:32]:
```suggestion
                    [here](https://discord.com/developers/docs/topics/gateway#list-of-intents).
```

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+      options:
+      - label: I double checked that I have the required intents.
+        required: true
+  - type: textarea
+    id: description
+    attributes: 
+      label: Description
+      description: A breif explination of the bug.
+      placeholder: When I start a DiscordSocketClient without stopping it the gatewat thread gets blocked.
```
Comment (lhjt) [inline .github/ISSUE_TEMPLATE/bugreport.yml:41]:
```suggestion
      description: A brief explination of the bug.
      placeholder: When I start a DiscordSocketClient without stopping it, the gateway thread gets blocked.
```

File: .github/ISSUE_TEMPLATE/bugreport.yml [hunk 6]
```diff
@@ -0,0 +1,77 @@
...
+      label: Working Version
+      description: If this worked on an older version of Discord.Net put that version here.
+      placeholder: ex. 2.4.0
+    validations:
+      required: false
+  - type: textarea
+    id: logs
+    attributes: 
+      label: Logs
+      description: Put applicable logs and/or a stacktrace here.
```
Comment (lhjt) [inline .github/ISSUE_TEMPLATE/bugreport.yml:64]:
```suggestion
      description: Add applicable logs and/or a stacktrace here.
```

**Review by lhjt — CHANGES_REQUESTED** [review]

**Comment by CottageDwellingCat** [comment]
a fixed version will be pred into labs shorty.

**Pull request closed without merging**